### ISSUE TITLE
Documentation Request - Provide flattened single-page listing of fields for easier search

### ISSUE THREAD

**Issue by AndrewMcQuerry** [elastic/ecs#608]
Sometimes when referring to the ECS documentation, we know what type of data we want to store, but not under which field set it would best fit.  As a result, we end up clicking through multiple doc pages, searching within each field set, then backing out and trying another one.

Providing a single-page view would make searching through the available fields much easier.. to more quickly decide which field set to use for a given "thing" to be stored.

**Comment by webmat** [comment]
Yes thanks for bringing this up, we're aware of that. We want this as well ;-)

As you already know from the workarounds I describe in #609, there's two ways to get useable.

We also are considering a redesign of the docs that may lend itself to having such a table directly in the official docs as well. It's mentioned in [this public brainstorm document](https://docs.google.com/document/d/1zIpQG8Flq1pVpV4TOGYolGfqP2lTGeVrxCbZkZsfmeo) somewhere in the table of things we'd like to improve. If you scroll down to the Annex section, you can see a prototype of what the new "field set" pages could look like. In short it would start with a more compact table, and be followed by a more fleshed out section about each field.

This more compact table could then be used to also offer an "all ECS fields" page.

Thoughts and comments welcome in there :-)
